### PR TITLE
Hdsrex 13

### PR DESCRIPTION
**Pull request by ushavemuganti-sharecare**
https://arnoldmedia.jira.com/browse/HDSREX-13

### Developer checkboxes

- [ ] I have checked the base branch is correct and filled in the github milestone appropriately
- [ ] I have reviewed my PR to ensure I followed established coding standards
- [ ] I have linked any other PRs for this ticket if there are any
- [ ] I have verified that the code meets the requirements of the ticket
- [ ] I have verified that there are tests defined for QA in the JIRA ticket and or these changes are covered by
  regression

### Reviewer Checkboxes

- [ ] The milestone, base branch and JIRA release versions all match
- [ ] Understand the ticket requirements and ensure the changes meet these requirements
- [ ] Check the tests in the Jira ticket fully test the changes
- [ ] Check for coding standards
- [ ] Consider how this change may affect other pieces of the system - capture these effects in the JIRA ticket
- [ ] Consider any worst case scenarios for these code changes - compare this to the risk in JIRA and update if
  necessary

### CODE REVIEW

**Comment by Harshitha1238-sharecare** [comment]
covered in another PR

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <link rel="icon" type="image/x-icon" href="assets/img/favicon.ico" />
         <link rel="preload" as="image" href="assets/img/sharecare-logo.webp" />
+        <link rel="stylesheet" href="https://unpkg.com/@progress/kendo-font-icons/dist/index.css" />
```
Comment (rajpeddalachugari) [inline src/index.html:10]: add it in angular.json

File: src/app/patient-request-type-form/patient-request-type-form.component.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    //     'Medications',
+    //     'Nurse Notes',
+    //     'Office Notes',
+    //     'Pathology Report',
+    //     'Physician Orders Report',
+    //     'Procedures/Operative Notes',
+    //     'Progress Notes',
+    //     'Therapy Notes',
+    //     'Other (Entire Chart, Actual Radiology Images, Billing Records, Specific Procedures, etc)',
+    // ];
```
Comment (rajpeddalachugari) [inline src/app/patient-request-type-form/patient-request-type-form.component.ts:56]: remove commented code if not required.

File: src/app/patient-request-type-form/patient-request-type-form.component.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        InputsModule,
+        ButtonsModule,
+        CommonModule,
+        TooltipModule,
+        IconsModule,
+    ],
+    templateUrl: './patient-request-type-form.component.html',
+    styleUrl: './patient-request-type-form.component.scss',
+})
+export class PatientRequestTypeFormComponent {
```
Comment (rajpeddalachugari) [inline src/app/patient-request-type-form/patient-request-type-form.component.ts:29]: implement onInit lifecycle hook to fix lint issues.

File: src/app/patient-request-type-form/patient-request-type-form.component.ts [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    @Input() patientRequestTypeForm!: FormGroup;
+    @Input() public listItems: any;
+    public flipping: IconFlip[] = ['default', 'horizontal', 'vertical', 'both'];
```
Comment (rajpeddalachugari) [inline src/app/patient-request-type-form/patient-request-type-form.component.ts:32]: do we need to flip icon ?

File: src/app/patient-request-type-form/patient-request-type-form.component.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+                    <div
+                        kendoTooltip
+                        style="text-align: center"
+                        [tooltipWidth]="250"
+                        *ngIf="item === 'Abstract Record Set'">
+                        <kendo-icon
+                            size="large"
+                            themeColor="info"
+                            flip="vertical"
```
Comment (rajpeddalachugari) [inline src/app/patient-request-type-form/patient-request-type-form.component.html:22]: flip attribute is added two times.

**Review by rajpeddalachugari — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (ushavemuganti-sharecare) [reply src/index.html]: removed the link from index.html and added input in anglar.json

File: src/app/patient-request-type-form/patient-request-type-form.component.ts (hunk 2 above)
Comment (ushavemuganti-sharecare) [reply src/app/patient-request-type-form/patient-request-type-form.component.ts]: removed the commented lines

File: src/app/patient-request-type-form/patient-request-type-form.component.ts (hunk 3 above)
Comment (ushavemuganti-sharecare) [reply src/app/patient-request-type-form/patient-request-type-form.component.ts]: removed type any and implemented onInit lifecycle hook

File: src/app/patient-request-type-form/patient-request-type-form.component.ts (hunk 4 above)
Comment (ushavemuganti-sharecare) [reply src/app/patient-request-type-form/patient-request-type-form.component.ts]: added the flip default in html and removed the array from ts file

File: src/app/patient-request-type-form/patient-request-type-form.component.html (hunk 5 above)
Comment (ushavemuganti-sharecare) [reply src/app/patient-request-type-form/patient-request-type-form.component.html]: removed one flip attribute

**Review by rajpeddalachugari — APPROVED** [review]

**Pull request merged**